jest.config.ts

```diff
@@ -1,14 +1,17 @@
-import type { InitialOptionsTsJest } from 'ts-jest/dist/types';
+import type { JestConfigWithTsJest } from "ts-jest";
```

Comment:
quote changes. 🙈

Reply:
Ah, I don't think prettier is enforced in this repo 🤦